sgkit/io/vcfzarr_reader.py

```diff
@@ -169,7 +169,7 @@ def _vcfzarr_to_dataset(
     if "variants/ID" in vcfzarr:
         variants_id = da.from_zarr(vcfzarr["variants/ID"]).astype(str)
     else:
-        variants_id = None
```

Comment:
Not worth testing this one?

Reply:
Refactored to cover this case (see https://github.com/pystatgen/sgkit/pull/381/commits/be7f5beff41973ef69dd60d90bd2ef3130a891bd)